net-libs/zeromq/zeromq-4.2.3.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="A lightweight messaging kernel"
+DESCRIPTION="The ZeroMQ lightweight messaging kernel is a library which \
+extends the standard socket interfaces with features traditionally provided by\
+ specialised messaging middleware products. ZeroMQ sockets provide an \
+abstraction of asynchronous message queues, multiple messaging patterns, \
+message filtering (subscriptions), seamless access to multiple transport \
+protocols and more."
+HOMEPAGE="http://www.zeromq.org/"
+COPYRIGHT="2007-2014 iMatix Corporation
+	2009-2011 250bpm s.r.o.
+	2010-2011 Miru Limited
+	2011 VMware, Inc.
+	2012 Spotify AB
+	2013 Ericsson AB
+	2014 AppDynamics Inc.
+	2015 Google, Inc.
+	2015-2016 Brocade Communications Systems Inc."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/zeromq/libzmq/releases/download/v$portVersion/zeromq-$portVersion.tar.gz"
+CHECKSUM_SHA256="8f1e2b2aade4dbfde98d82366d61baef2f62e812530160d2e6d0a5bb24e40bc0"
+SOURCE_URI_2="https://raw.githubusercontent.com/zeromq/cppzmq/a604af9/zmq.hpp#noarchive"
+CHECKSUM_SHA256_2="02b73219656df3ffe3b231c7d0326e754a738ef2a83c8e94cd1e47d22cdd9535"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64" 
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```
In `SECONDARY_ARCHITECTURES` we don't need to mention those secondary arches which are already marked as broken in `ARCHITECTURES`.
Actually, I think it is even recommended to not mention `!x86_gcc2` in `SECONDARY_ARCHITECTURES`. Having `!x86_gcc2` in `ARCHITECTURES` is sufficient.

BTW, if zeromq can't be easily built on x86_gcc2 primary arch then it might be a good idea to define [`commandBinDir`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable) (and `commandSuffix`) and replace `cmd:curve_keygen$secondaryArchSuffix` by `cmd:curve_keygen$commandSuffix`.)

Reply:
checked the build for x86_gcc2 and it failed yes (even with some small changes), I'll set commandSuffix :)